policy/cultural-data-exchange-policies.ttl

```diff
@@ -0,0 +1,289 @@
+@prefix drk: <https://w3id.org/drk/ontology/> .
+@prefix drkr: <https://w3id.org/drk/resource/> .
+@prefix void: <http://rdfs.org/ns/void#> .
+@prefix dcat: <http://www.w3.org/ns/dcat#> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix odrl: <http://www.w3.org/ns/odrl/2/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+
+###############################################
+#Cultural Data Exchange Policies
+###############################################
+
+###############################################
+# Scenario 1: Cultural API Access Management
+###############################################
+# Scenario Description:
+# - The Augsburg State Theater provides a ShowTimes API
+# - Access is restricted geographically to Germany
+# - Time-limited access until May 10, 2025
+# - Three types of subscribers have access rights
+# - Access is managed through a TempoSpatialAccess policy
+
+
+# ShowTimes API Asset
+drkr:DE_Augsburg_STAugsburg_ShowtimesAPI a drk:ShowTimesAPI, odrl:Asset ;
```

Comment:
Where is the class `drk:ShowTimesAPI` defined? (I can't find it, neither in this PR nor in the previously existing ontology.)